partials/google-analytics.hbs

```diff
@@ -1,11 +1,9 @@
 <script>
-    if (window.ga_id) {
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-        ga('create', window.ga_id, 'auto');
-        ga('require', 'linkid', 'linkid.js');
-        ga('send', 'pageview');
-    }
-</script>
+(function(e,t,n,i,s,a,c){e[n]=e[n]||function(){(e[n].q=e[n].q||[]).push(arguments)}
+;a=t.createElement(i);c=t.getElementsByTagName(i)[0];a.async=true;a.src=s
+;c.parentNode.insertBefore(a,c)
+})(window,document,"galite","script","https://cdn.jsdelivr.net/npm/ga-lite@2/dist/ga-lite.min.js");
+
+galite('create', 'UA-57914210-1', 'auto');
```

Comment:
This google_analytics id is hard coded and we actually want to allow the consumer of this theme to be able to custom define/use their own id. 

So can you make the following changes:

1. Wrap this entire code change with the same `if (window.ga_id)` check that I made before. 
2. Replace `galite('create', 'UA-57914210-1', 'auto');` with `galite('create', 'UA-window.ga_id', 'auto');`

Reply:
I've updated the changes to the code so that a consumer can define their own GA code from the frontend. :)

This has been pushed to my latest commit.